test/functionalTest/cases/0000_https_support/https.test

```diff
@@ -27,7 +27,10 @@ HTTPS simple test
 dbInit CB
 ${SCRIPT_HOME}/httpsPrepare.sh --keyFileName /tmp/harnessTest.key --certFileName /tmp/harnessTest.pem
 extraParams="-https -key /tmp/harnessTest.key -cert /tmp/harnessTest.pem"
+
+export BROKER_AWAIT_SLEEP_TIME=6
 brokerStart CB 0-255 IPV4 "$extraParams"
+unset BROKER_AWAIT_SLEEP_TIME
 
```

Comment:
Some functests fail "exit code 11" under valgrind if the broker is started with the new method of using nc and awaiting the broker that way.
I have fixed this problem by adding this alternate way - which is the old way, just a normal sleep (but with $BROKER_AWAIT_SLEEP_TIME instead of a hardcoded 10 seconds)